tests/foreman/cli/test_oscap.py

```diff
@@ -75,13 +98,14 @@ def test_positive_list_default_content_with_admin(self):
 
         :expectedresults: The scap-content are listed.
 
-        :caseautomation: notautomated
+        :caseautomation: automated
 
         :CaseImportance: Critical
         """
+        result = Scapcontent.list()
+        self.assertGreater(len(result), 0)
```

Comment:
Is there any default name/value, presence of which we can check? I'm afraid we can generate false positive here when empty string is returned - list with 1 item (empty string) will pass this check.